onnxruntime/core/framework/session_state_initializer.h

```diff
@@ -12,44 +12,14 @@
 #include "core/platform/path_lib.h"
 
 namespace onnxruntime {
-class ExecutionProviders;
-class Graph;
-class GraphTransformerManager;
-class InsertCastTransformer;
 class KernelRegistryManager;
 class Node;
-class NodeArg;
 class SessionState;
 
-namespace logging {
-class Logger;
-}
+Status FinalizeSessionState(SessionState& session_state,
```

Comment:
nit: should the filename change to reflect the new changes?

Reply:
I left it as is so the diff was easier, but will change it now. 

---
In reply to: [443774965](https://github.com/microsoft/onnxruntime/pull/4194#discussion_r443774965) [](ancestors = 443774965)